pkgs/by-name/id/ida-free/package.nix

```diff
@@ -107,31 +80,38 @@ stdenv.mkDerivation rec {
     runHook preInstall
 
     mkdir -p $out/bin $out/lib $out/opt
+    mkdir -p $out/.local/share/applications
 
     # IDA depends on quite some things extracted by the runfile, so first extract everything
     # into $out/opt, then remove the unnecessary files and directories.
     IDADIR=$out/opt
 
+    # Desktop
+    HOME=$out
```

Comment:
Maybe add a comment here that the installer doesn't honor `--prefix` in all places, thus needing to set `HOME` here.